__tests__/api-endpoints.test.js

```diff
@@ -140,25 +140,25 @@ describe("/api/articles/:article_id/comments - happy path", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
-      .then(({ body: comment }) => {
-        expect(comment.comments.length).toBe(11);
+      .then(( comment ) => {
+        expect(comment.body.comment.length).toBe(11);
```

Comment:
I'll give you a trick for this 

`.then( ({ body: { comments }}) => {`

then you can access `comments` directly without any dot notation - I think yours is named `comment` without the `s` but it should have the `s` because it's an array